src/index.ts

```diff
@@ -219,4 +220,68 @@ app.get("/subjects/:subject_id", async (c) => {
     ),
   });
 });
+
+function sinceAndUntilSetting(since: string = '0001-01-01T00:00:00', until: string = '9999-12-31T23:59:59'): { secondsSince: number; secondsUntil: number; } | null {
+  const msSince = Date.parse(since);
+  const msUntil = Date.parse(until);
+  if ( Number.isNaN(msSince) || Number.isNaN(msUntil) ){
+    return null;
+  }
+  const secondsSince = Math.floor(msSince / 1000);
+  const secondsUntil = Math.floor(msUntil / 1000);
+
+  return { secondsSince, secondsUntil }
+}
+
+app.get("/subject/:subject_id/:board_id/attendances", async (c) => {
+  //各パラメータの受け取り
+  const { since, until }= c.req.query();
+  const boardId = c.req.param("board_id");
+
+  //認証する
+  const HonoSession = c.get("session");
+  const login = HonoSession.get("login") as Session | null;
```

Comment:
```suggestion
  const login = c.get("login");
```
現行の main ブランチのコードはこれだけでログイン情報を取得できるようになっているので, このようにお願いします.